src/vcpkg/export.prefab.cpp

```diff
@@ -485,9 +489,17 @@ namespace vcpkg::Export::Prefab
 
             if (prefab_options.enable_debug)
             {
-                print2(Strings::format("[DEBUG]\n\tWriting manifest\n\tTo %s\n\tWriting prefab meta data\n\tTo %s\n\n",
-                                       manifest_path,
-                                       prefab_path));
+                msg::println(msg::format(msgWritingManifest)
+                                 .append_raw("\n")
```

Comment:
```suggestion
                                 .append_raw('\n')
```

Quick perf tip: Saves call to `strlen`. Same elsewhere.